### PR TITLE
Fixed some bugs.

### PR DESCRIPTION
**Pull request by ankushbhagats**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ addrepo() {
 }
 
 echo "[i] Installing TermuXBlacK ..."
-printf "Are you want TermuxBlack Custom PS1? [Y/n] "
+printf "[?] Do you want TermuxBlack Custom PS1? [Y/n] "
```
Comment (trukkijarno-dot) [inline install.sh:38]: Y

Comment (trukkijarno-dot) [inline install.sh]: Y

**Review by trukkijarno-dot — APPROVED** [review]

File: install.sh (hunk 1 above)
Comment (mokninedjamal986-netizen) [reply install.sh:38]: https://github.com/features/ai/github-app?utm_source=mobile-copilot-app&utm_medium=product&utm_campaign=github-app-cli-Q1-push

Comment (trukkijarno-dot) [reply install.sh:38]: > sorry i dont know who is using my account def not me whos messing around 👎🏽 :(

**Review by trukkijarno-dot — COMMENTED** [review]
if someone can help me with hacker problem please reach out..

**Pull request merged**